modules/users/client/config/users.client.routes.js

```diff
@@ -101,20 +101,6 @@
         url: '/reset',
         template: '<ui-view/>'
       })
-      .state('password.reset.invalid', {
```

Comment:
@mleanos really good feedback, appreciate it! 

I redirected the success to home with a toast. Looks like I missed the error route. Honestly I haven't see these flows in action as I don't have a MSP. Can someone recommend some simple free MSP so I can test these flows?

I simply removed those routes and deleted those two view files to reduce footprint. In those terms I could have the invalid scenario redirect to forgot password screen with an invalid/expired token toast so end-user can request another.

Or the opposite being what you recommend to keep the routes and views in case users need them.

I am neutral to both scenarios. Which one do you prefer?